getting-started.md

```diff
@@ -171,9 +171,9 @@ but you can deploy Flutter apps to iOS devices.
 - Run `Simulator.app`
   - You can find Simulator.app in
 _Applications/Xcode/Contents/Developer/Applications_ or via _Spotlight_
-  - To ensure your simulator uses a device type of iPhone 5 or above,
+  - To ensure your simulator uses a device type of iPhone 5s or above,
```

Comment:
I would explictly mention 64-bit here.

"Ensure your simulator is using a 64-bit device (iPhone 5s or later)."